src/NServiceBus.ContainerTests/When_building_components.cs

```diff
@@ -41,14 +41,17 @@ public void Singlecall_components_should_yield_unique_instances()
         }
 
         [Test]
-        public void UoW_components_should_resolve_from_main_container()
+        public void UoW_components_should_yield_the_same_instance()
```

Comment:
do we have a test that the instances from the child container are different than the one from the parent?